### PR TITLE
Initial commit of vocabulary files.

### LINKED ISSUE #33: RDFS / OWL for VC vocabulary?
Hello, I'm curious if there are plans to define an RDF schema or OWL ontology for the VC vocabulary. E.g. what would be the rdfs:domain and rdfs:range of the cred:claim property?

### PR DESCRIPTION
**Pull request by gkellogg**
This simply adds the vocabulary updates described in the F2F at TPAC. Ill file issues on issues raised at TPAC.

Fixes #33.

### CODE REVIEW

File: vocab/credentials.jsonld [hunk 1]
```diff
@@ -0,0 +1,251 @@
+{
```
Comment (msporny) [inline vocab/credentials.jsonld]: Here's the latest context from the test suite: https://github.com/w3c/vc-test-suite/blob/gh-pages/contexts/vc-v1.jsonld

File: vocab/credentials.jsonld [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    },
+    "revoked": {
+      "@id": "sec:revoked",
+      "@type": "xsd:dateTime"
+    },
+    "signature": "sec:signature",
+    "signatureAlgorithm": "sec:signatureAlgorithm",
+    "signatureValue": "sec:signatureValue",
+    "Credential": "cred:Credential",
+    "claim": {
```
Comment (msporny) [inline vocab/credentials.jsonld]: This needs to be updated in the test suite and here - "@container": "@graph", I think. @dlongley, I forget the details, but it looks like it's wrong in both the test suite and here.

File: vocab/credentials.jsonld [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    },
+    "recipient": {
+      "@id": "cred:recipient",
+      "@type": "@id"
+    },
+    "referenceId": {
+      "@id": "cred:referenceId"
+    }
+  },
+  "@graph": {
```
Comment (msporny) [inline vocab/credentials.jsonld]: I'm wondering if we should split this out, where doing https://w3id.org/credentials gets this document, and https://w3id.org/credentials/v1 just gets the context file w/o the RDFs/OWL vocabulary description. I'm concerned about file size, but perhaps unnecessarily so. I know that we always cache local copies in our programs and almost never go out to the network to get known contexts.

Comment (gkellogg) [reply vocab/credentials.jsonld]: I'll update the script to create both the all-inclusive vocabulary and a separate context-only file.

File: vocab/credentials.jsonld (hunk 2 above)
Comment (gkellogg) [reply vocab/credentials.jsonld]: I'll add it, but we haven't defined `@container: @graph` in the spec, yet.

**Comment by gkellogg** [comment]
Updated per @msporny's suggestions. There are some things that may still require abstraction (e.g., a super-class for `RevocationList2017`.

**Comment by gkellogg** [comment]
@msporny Do these changes satisfy your review? Can you approve it?

**Comment by msporny** [comment]
@gkellogg LGTM. @dlongley can you review please?

File: vocab/credentials.jsonld [hunk 4]
```diff
@@ -246,6 +301,18 @@
           "en": "FIXME"
         }
       }
+    ],
+    "rdfs_instances": [
+      {
+        "@id": "cred:revocation",
+        "@type": "rdf:PRoperty",
```
Comment (msporny) [inline vocab/credentials.jsonld]: Typo rdf:PRoperty

File: vocab/context.jsonld [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+      "@id": "dc:creator",
+      "@type": "@id"
+    },
+    "domain": "sec:domain",
+    "expires": {
+      "@id": "sec:expiration",
+      "@type": "xsd:dateTime"
+    },
+    "nonce": "sec:nonce",
+    "normalizationAlgorithm": "sec:normalizationAlgorithm",
```
Comment (dlongley) [inline vocab/context.jsonld]: I believe this has been renamed to `canonicalizationAlgorithm` (both terms appear in the security vocabulary but we decided to move forward with `canonicalizationAlgorithm`). Should I file a separate issue for this or should we just change it here?

File: vocab/context.jsonld [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    "CryptographicKeyCredential": "cred:CryptographicKeyCredential",
+    "Profile": "cred:Profile",
+    "RevocationList2017": "cred:RevocationList2017",
+    "claim": {
+      "@id": "cred:claim",
+      "@container": "@graph"
+    },
+    "credential": {
+      "@id": "cred:credential",
+      "@type": "@id"
```
Comment (dlongley) [inline vocab/context.jsonld]: The `credential` field should also be: `"@container": "@graph"`. Again, should I file an issue for these sorts of issues separately or should we be making the changes now in this PR?

File: vocab/context.jsonld (hunk 5 above)
Comment (msporny) [reply vocab/context.jsonld]: Let's make the change in this PR.

File: vocab/context.jsonld (hunk 6 above)
Comment (msporny) [reply vocab/context.jsonld]: Let's make the change in this PR... these are "small" changes.

File: vocab/credentials.jsonld (hunk 4 above)
Comment (gkellogg) [reply vocab/credentials.jsonld]: Got it.

File: vocab/context.jsonld (hunk 5 above)
Comment (gkellogg) [reply vocab/context.jsonld]: 👍

File: vocab/context.jsonld (hunk 6 above)
Comment (gkellogg) [reply vocab/context.jsonld]: 👍

**Comment by gkellogg** [comment]
I made the three suggested changes. Note that the procedure is to update the vocab.csv file and re-run mk_vocab.rb, rather than editing generated documents by hand.

**Comment by msporny** [comment]
+1 to merge, I'll request a merge at the next telecon.

**Pull request merged**